README.md

```diff
@@ -109,6 +109,8 @@ Have any feedback or questions? [Create a discussion](https://github.com/TwiN/ga
   - [API](#api)
   - [Installing as binary](#installing-as-binary)
   - [High level design overview](#high-level-design-overview)
+- [Troubleshooting](#troubleshooting)
+  - [How to solve `Request Header Fields Too Large` error?](#how-to-solve-request-header-fields-too-large-error)
```

Comment:
I feel like there's a bit of an overlap between `Troubleshooting` and the current use of the `FAQ` section 🤔